Dockerfile.arm64

```diff
@@ -0,0 +1,20 @@
+
+FROM alpine:3.6 as alpine
+RUN apk add -U --no-cache ca-certificates mailcap
+
+FROM scratch
+MAINTAINER Drone.IO Community <drone-dev@googlegroups.com>
+
+ENV GODEBUG=netdns=go
+
+COPY --from=alpine /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+COPY --from=alpine /etc/mime.types /etc/
```

Comment:
This is not required, same for the mailcap package above

Reply:
you mean the mime.types and the certs? or just the mime.types.  Sorry I actually have not idea what they're for :)

Reply:
Just the mime.types should be dropped, not required for this plugin :)